MantidQt/CustomInterfaces/src/Reflectometry/ReflMainWindowPresenter.cpp

```diff
@@ -34,35 +34,41 @@ ReflMainWindowPresenter::ReflMainWindowPresenter(
 ReflMainWindowPresenter::~ReflMainWindowPresenter() {}
 
 /** Returns global options for 'CreateTransmissionWorkspaceAuto'
+*
+* @return :: Index of the group in 'Settings' tab from which to get the options
```

Comment:
Shouldn't this be `@param`? I'm surprised that Doxygen didn't pick up on this though. The same goes for all other methods in this class.